lib/queue.js

```diff
@@ -1,27 +1,59 @@
 class Queue {
+    #front;
+    #rear;
+
     constructor() {
-        // this.store = ...
-        throw new Error("This method has not been implemented!");
+        this.store = new Array(20);
```

Comment:
👀 When implementing a circular buffer, it's often helpful to have a `capacity` field to enable users to set the size of the buffer (you'd want to set `capacity` as a parameter which can to default to 20)
```suggestion
        this.capacity = capacity
        this.store = new Array(capacity);
```